collections/triple.go

```diff
@@ -275,29 +275,44 @@ func (t tripleKeyCodec[K1, K2, K3]) SizeNonTerminal(key Triple[K1, K2, K3]) int
 
 // NewPrefixUntilTripleRange defines a collection query which ranges until the provided Pair prefix.
 // Unstable: this API might change in the future.
-func NewPrefixUntilTripleRange[K1, K2, K3 any](k1 K1) Ranger[Triple[K1, K2, K3]] {
+func NewPrefixUntilTripleRange[K1, K2, K3 any](k1 K1, reverse bool) Ranger[Triple[K1, K2, K3]] {
```

Comment:
Can we not break the API, please? We strive to have more stable APIs.
Additionally, I am super a fan of just having a bool in the function.

Can create new APIs instead? Like `NewPrefixUntilTripleRangeReversed` or something alike.

Reply:
Sure.
I thought maybe turn it to a variadic function so the reverse bool will be optional. By doing that, the API won't break but will be able to support reverse iteration.
Your solution obviously works too, but it add 3 functions that are basically the same code but with the "order" field set to reverse which is a sort of code duplication

Reply:
Adding options would work too yes 👍🏾.

Reply:
done. see https://github.com/cosmos/cosmos-sdk/pull/21496/commits/1b1d1a7652cf18a9b1d458c8c2526a7432e1c6b8